applications.yaml

```diff
@@ -642,6 +642,14 @@
     id: '#32770'
     comment: Dialog Box
     matching_strategy: Legacy
+- name: Microsoft Edge
+  identifier:
+    kind: Exe
+    id: msedge.exe
+    matching_strategy: Equals
+  options:
+  - tray_and_multi_window
+  - object_name_change
```

Comment:
I don't think `object_name_change` is necessary here

Reply:
yes, edge and teams are not always sending the ShowObject event, and then komorebi fails to see the window. Franckly I am not sure the object_name_change trick is so good.
I think we should just call ShowObject if we see a visible window that is yet unknown, but this is beyond the scope of this simple PR, and I don't have all your experience on these trickeries..